api/v1/models/user.py

```diff
@@ -30,6 +30,7 @@ class User(BaseModel, Base):
     email = Column(String(100), unique=True, nullable=False)
     password = Column(String(255), nullable=False)
     first_name = Column(String(50))
+    phone_number = Column(String(25), nullable=True)
     last_name = Column(String(50))
```

Comment:
Hi, @Dev-wonderful I've been working on the corrections. Yes, I have a migration file but it's not pushed because the `alembic/versions` folder was added to `.gitignore`. I don't know if I'm allowed to change that.